.circleci/build-assets.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+echo "Begin build for $CIRCLE_ENV${PR_ENV:+ for }$PR_ENV. Pantheon test environment is $TERMINUS_SITE.$TERMINUS_ENV"
+
+# Build assets with Composer
+composer -n build-assets
+
+# Compile Sass or run any other build steps necessary
```

Comment:
For Drupal 8, theme information is in the guide: https://pantheon.io/docs/guides/github-pull-requests/#create-a-custom-theme

We will need a guide for Drupal 7 and WordPress as well; we could add some Sass advice there.